R/gg-plots.R

```diff
@@ -320,25 +388,26 @@ ggally_cor <- function(
     }
 
     if (any(!rows)) {
+      if (!is.null(colorData) && (length(colorData) == length(xData))) {
+        colorData <- colorData[rows]
+      }
+      xData <- xData[rows]
+      yData <- yData[rows]
+
       total <- sum(!rows)
       if (total > 1) {
         warning("Removed ", total, " rows containing missing values")
       } else if (total == 1) {
         warning("Removing 1 row that contained a missing value")
       }
     }
-
-    if (!is.null(colorData) && (length(colorData) == length(xData))) {
-      colorData <- colorData[rows]
-    }
-    xData <- xData[rows]
-    yData <- yData[rows]
   }
 
   xVal <- xData
   yVal <- yData
 
   # if the mapping has to deal with the data, remove it
+  ### IDK what this does
   if (packageVersion("ggplot2") > "2.2.1") {
```

Comment:
Could be simplified. Not sure it is still relevant to support old versions of ggplot2. In other PR I sent, I added ggplot2 >= 3.3.0 because of the change in the way you access to computed variables in a sat

Reply:
I'd be very happy to remove old ggplot2 support.

Will do